tests/python/relay/test_auto_scheduler_tuning.py

```diff
@@ -23,6 +23,15 @@
 from test_auto_scheduler_task_extraction import get_network
 
 
+class CustomMeasureCallback(auto_scheduler.measure.PythonBasedMeasureCallback):
```

Comment:
It is better to move this test to here (https://github.com/apache/tvm/blob/e51bcdd1ed99fc813454c68911f8032c852f48b4/tests/python/unittest/test_auto_scheduler_search_policy.py#L71).

Because this measure callback is not related to relay. We should not put it under `tests/python/relay/`

Reply:
Moved.